data/src/main/java/co/nimblehq/compose/crypto/data/repository/CoinRepositoryImpl.kt

```diff
@@ -30,4 +32,20 @@ class CoinRepositoryImpl(
             page = page
         )
     }.map { it.toModels() }
+
+    override fun getTrends(
```

Comment:
I'd suggest renaming this a bit, so we will have `getCoins, getTrendingCoins` with more context and meaning.

```suggestion
    override fun getTrendingCoins(
```

Reply:
actually we can even reuse `getCoin` method for both usecases, don't need to define 1 more 🤔

Reply:
@luongvo Get trending coins doesn't require `ids` on the parameter list

```
interface ApiService {
    @GET("coins/markets")
    suspend fun getCoins(
        @Query("ids") coinIds: String,
        @Query("vs_currency") currency: String,
        @Query("price_change_percentage") priceChangePercentage: String,
        @Query("order") itemOrder: String,
        @Query("per_page") itemPerPage: Int,
        @Query("page") page: Int
    ): Response<List<CoinItemResponse>>

    @GET("coins/markets")
    suspend fun getTrends(
        @Query("vs_currency") currency: String,
        @Query("price_change_percentage") priceChangePercentage: String,
        @Query("order") itemOrder: String,
        @Query("per_page") itemPerPage: Int,
        @Query("page") page: Int
    ): Response<List<TrendingItemResponse>>
}
```

Reply:
@hoangnguyen92dn yes, @Wadeewee can we skip sending it by `null` param?

Reply:
@luongvo [here](https://github.com/nimblehq/jetpack-compose-crypto/pull/26#discussion_r963222360) my idea